sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Helpers/ContainerRepository.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Containers.ContainerRegistry
```

Comment:
Nit: while the namespace fortunately doesn't reflect this, seems odd to have this public class in a directory named "Helpers". If I were onboarding, it's not where I'd expect to look for anything public.

Reply:
That's a good point - I'll change it!